utils/csv_2_json_data_converter.py

```diff
@@ -0,0 +1,171 @@
+                        os.listdir(self.data_dir) if os.path.isfile(os.path.join(self.data_dir, file)) and
+                        os.path.splitext(file)[-1] == most_common_extension and
+                        file.split('.')[0] == most_common_prefix]
+
+        print(f"{bcolors.OKMSG}Files for conversion :{bcolors.END}")
+        for file in dir_list:
+            print(f"{bcolors.OKMSG}=>{bcolors.END} {file[0]}")
+
+        return dir_list
+
+    def _convert_file(self, file_path: tuple) -> None:
+        """ Method converts a file and saves converted file to given path
+        """
+        file_to_convert = file_path[0]
+        node = int(os.path.split(file_to_convert)[-1].split('.')[-2])
+        file_to_save = file_path[1]
+        read_file_list = self._read_csv(file_to_read=file_to_convert)
+        read_file_dict = self._get_data_phase_sorted(data=read_file_list)
+        self._write_json(output_path=file_to_save, data_to_convert=read_file_dict, node=node)
+
+    @staticmethod
+    def _read_csv(file_to_read: str) -> list:
+        """ Method reads csv and returns a list of dicts (phase, object_id, time) ready to save into JSON
+        """
+        with open(file_to_read, 'rt') as csv_file:
+            log = csv.reader(csv_file, delimiter=',')
+            read_list = [{'phase_id': int(row[0]), 'obj_id': int(row[1]), 'obj_time': float(row[2])} for row in log
+                         if len(row) == 3]
+
+        return read_list
+
+    @staticmethod
+    def _get_data_phase_sorted(data: list) -> dict:
+        """ Method sorts data with respect to the phase. Returns dict with phases as keys
+        """
+        # Creating temporary dict, so rows are sorted by phase_id
+        temp_dict = dict()
+        for dict_ in data:
+            phase_id = dict_.get('phase_id', None)
+            obj_id = dict_.get('obj_id', None)
+            obj_time = dict_.get('obj_time', None)
+            if isinstance(temp_dict.get(phase_id, None), list):
+                temp_dict[phase_id].append({'phase_id': phase_id, 'obj_id': obj_id, 'obj_time': obj_time})
+            else:
+                temp_dict[phase_id] = list()
+                temp_dict[phase_id].append({'phase_id': phase_id, 'obj_id': obj_id, 'obj_time': obj_time})
+
+        return temp_dict
+
+    def _write_json(self, output_path: str, data_to_convert: dict, node: int) -> None:
+        """ Method converts data to JSON and saves to output path
+        """
+        # Creating dictionary with right structure, which will be dumped
+        dict_to_dump = dict()
+        dict_to_dump['phases'] = list()
+        for proc_id, others_list in data_to_convert.items():
+            phase_dict = {'tasks': list(), 'id': proc_id}
+            for task in others_list:
+                task_dict = {'time': task['obj_time'], 'resource': 'cpu',
+                             'entity': {'id': task['obj_id'], 'type': 'object'}, 'node': node}
```

Comment:
@lifflander Do we want to omit `home` or reverse engineer the entity ID (differently for CSVs that come from the release branch vs develop) so that the data is more complete?  With the `home` specified in all stats data imported into my LB testing/tuning driver in vt, I can delete a ton of code, but if we want to maintain compatibility with data that lacks `home` then I will need to retain all that extra functionality.